mapswipe_workers/tests/unittests/fixtures/tile_map_service_grid/users/users_-M7nTGflMusNxFd4iSfD.csv

```diff
@@ -1,247 +1,247 @@
-221,-M7nTGflMusNxFd4iSfD,uV5Xsx4DTgYtrAoRqtGreblCbch2,2,582,849,606,0.5835051546391753
-222,-M7nTGflMusNxFd4iSfD,ur9fvMTKXAOYplEfiV8ob8eFKw83,4,1164,2013,315,0.8646907216494846
-223,-M7nTGflMusNxFd4iSfD,vWdSAsvXoOVhpp0UAl3ctfraCmZ2,1,291,279,303,0.4793814432989691
-224,-M7nTGflMusNxFd4iSfD,vrsCjVBjHucHS2E6sJ5Q40O7VO92,2,579,558,600,0.48186528497409326
-225,-M7nTGflMusNxFd4iSfD,vsJkKbe3lnTj6duL2rQkz6TAP2s2,1,291,281,301,0.48281786941580757
-226,-M7nTGflMusNxFd4iSfD,wGUTES1ADjMeCUUmAqLHeFxipKA3,1,291,582,0,1.0
-227,-M7nTGflMusNxFd4iSfD,wORe4LkQKObfaEGLgFES3lGaRtk1,2,579,579,579,0.5
-228,-M7nTGflMusNxFd4iSfD,wOhcxzDUE4S8zGyK5ZfuFe8weyN2,1,288,298,278,0.5173611111111112
-229,-M7nTGflMusNxFd4iSfD,wUL1amJSEdbnREq21tfmZNDkwYh1,83,24075,39442,12194,0.7638469284994964
-230,-M7nTGflMusNxFd4iSfD,wjxfCZUPnXRtuQ498XajlFMJS1J3,1,288,542,34,0.9409722222222222
-231,-M7nTGflMusNxFd4iSfD,wtymC6kNjKOMp1z6sX77Vb1Omdq2,1,291,311,271,0.5343642611683849
-232,-M7nTGflMusNxFd4iSfD,xp6ustWT20UBipgs3yelxgmZYXn1,5,1446,2554,338,0.8831258644536653
-233,-M7nTGflMusNxFd4iSfD,y9GOTBtqx3eopYYt4BxsPYk8XFu2,65,18819,36519,1119,0.9702694085764387
-234,-M7nTGflMusNxFd4iSfD,yOs4waU5GBZ5UvEHAP0rcEmZ1bY2,17,4932,8964,1191,0.8827178729689807
-235,-M7nTGflMusNxFd4iSfD,yXvZL0rHITUV7qz4BXiEyn6Zxdy2,2,579,1071,666,0.616580310880829
-236,-M7nTGflMusNxFd4iSfD,yY6lac2lWgelLqQgN5YY2JOWWbu1,23,6669,10892,3610,0.751068818094056
-237,-M7nTGflMusNxFd4iSfD,yub1C6sWfpa2hN3cQT6pH4rkbwv1,6,1740,4177,1334,0.7579386681183088
-238,-M7nTGflMusNxFd4iSfD,z1DsgPqIRAhRdCPy1KGyK3nnbb92,4,1158,2745,438,0.8623939679547596
-239,-M7nTGflMusNxFd4iSfD,z5NVCWWhSBNrrPNuxcBAO69QnUi1,11,3195,6009,2124,0.7388417558096644
-240,-M7nTGflMusNxFd4iSfD,zLkHJgxvLpYswoSePSeuP7oXb1U2,13,3759,3433,4373,0.4397899052011273
-241,-M7nTGflMusNxFd4iSfD,zRFa1hIM6tdci62SYhSAyXkG50e2,1,291,840,33,0.9621993127147767
-242,-M7nTGflMusNxFd4iSfD,zXdfy2pzcqa3LIWCraavPRbcV8h2,3,867,1368,366,0.7889273356401384
-243,-M7nTGflMusNxFd4iSfD,zs9IdLDJ0hfnWAHc9PF1NiMnX3k1,1,291,820,53,0.9392898052691867
-244,-M7nTGflMusNxFd4iSfD,ztgbNPjRjTUuOKlknwDgejvpxXo2,1,291,580,2,0.9965635738831615
-245,-M7nTGflMusNxFd4iSfD,zxS3YtVzToRSmBsKT6k5P2t837I2,3,867,1393,341,0.8033448673587081
+idx,project_id,user_id,username,groups_completed,total_contributions,agreeing_contributions,disagreeing_contributions,simple_agreement_score
+0,-M7nTGflMusNxFd4iSfD,08tsiIBXUNOBYtO1ZwmGU9t1iKg1,Test-user,7,2025,2769,1569,0.6383125864453665
+1,-M7nTGflMusNxFd4iSfD,0rluq7FANKRXsiNFNDRriq5PVZ72,Test-user,2,579,1151,7,0.9939550949913644
+2,-M7nTGflMusNxFd4iSfD,0sbtU3vn1rSvD7dY3zIbOO19bWy2,Test-user,3,867,1376,649,0.6795061728395062
+3,-M7nTGflMusNxFd4iSfD,13MrH4g49oRR1oyAjl0DfVw9KBQ2,Test-user,5,1452,2026,878,0.6976584022038568
+4,-M7nTGflMusNxFd4iSfD,1E0qG4OyDiXyhClCyQtQvCTpypD2,Test-user,23,6648,10834,3041,0.7808288288288289
+5,-M7nTGflMusNxFd4iSfD,1O6U2O6LTuhMHROAtBG9EcBqrJQ2,Test-user,1,288,788,76,0.9120370370370372
```

Comment:
This result is changed at the 16th decimal. I'm assuming other results change in a similar way. Is it important? Why does the result change? @Hagellach37 what do you think?

Reply:
I haven't checked the previous CSV file, as the diff is almost a million lines long :(

Reply:
username field was added there. Not sure why we have 448597 rows here.